configuration/cardano/mainnet-config-bp.json

```diff
@@ -12,9 +12,15 @@
   "LastKnownBlockVersion-Alt": 0,
   "LastKnownBlockVersion-Major": 3,
   "LastKnownBlockVersion-Minor": 0,
+  "LedgerDB": {
+    "Backend": "V2InMemory",
+    "NumOfDiskSnapshots": 2,
+    "QueryBatchSize": 100,
+    "SnapshotInterval": 4320
```

Comment:
For these three options, it is fine to make them explicit which would use something like `RequestedNumOfDiskSnapshots 2`, or leave them as undefined which will use `DefaultNumOfDiskSnapshots` which in the end will result in the same value.